apps/backoffice-v2/src/pages/Entities/components/CaseGeneration/components/CaseGenerationForm/CaseGenerationForm.tsx

```diff
@@ -0,0 +1,35 @@
+import { TWorkflowDefinition } from '@/domains/workflows/fetchers';
+import { SubmitSection } from '@/pages/Entities/components/CaseGeneration/components/CaseGenerationForm/components/SubmitSection';
+import { useCaseGenerationForm } from '@/pages/Entities/components/CaseGeneration/components/CaseGenerationForm/hooks/useCaseGenerationForm';
+import { useFormSchema } from '@/pages/Entities/components/CaseGeneration/components/CaseGenerationForm/hooks/useFormSchema';
+import { transformErrors } from '@/pages/Entities/components/CaseGeneration/components/CaseGenerationForm/utils/transformErrors';
+import { DynamicForm, baseLayouts } from '@ballerine/ui';
+import { FunctionComponent } from 'react';
+
+interface FormProps {
+  workflowDefinition: TWorkflowDefinition;
+}
+
+const layouts = {
+  ...baseLayouts,
+  ButtonTemplates: {
+    ...baseLayouts.ButtonTemplates,
+    SubmitButton: SubmitSection,
+  },
+};
```

Comment:
Might help with the type assertion below.
```suggestion
} satisfies typeof baseLayouts;
```